be/src/exec/pipeline/scan/chunk_source.h

```diff
@@ -66,6 +66,15 @@ class ChunkSource {
     void pin_chunk_token(ChunkBufferTokenPtr chunk_token);
     void unpin_chunk_token();
 
+    // for stream
```

Comment:
how about add stream chunk source?

Reply:
It is added already in `binlog_connector`